pkg/shared/families/malware/yara/yara.go

```diff
@@ -112,6 +128,27 @@ func (s *Scanner) Run(sourceType utils.SourceType, userInput string) error {
 			return
 		}
 
+		// If the (stdout lines)/(stderr + stdout lines) is less than `errThreshold` the error threshold will be reached.
```

Comment:
" If stderr lines / stderr lines + stdout lines is greater than the err threshold..."

Reply:
https://github.com/openclarity/vmclarity/pull/732/commits/7d454c033f9373ffe7bed7259fb80514240d9037